plugins/inputs/opcua_listener/README.md

```diff
@@ -0,0 +1,200 @@
+  # identifier_type =
+  #
+  ## Node ID Configuration.  Array of nodes with the same settings as above.
+  ## Extra tags to be added to the output metric
+  ##   deprecated in 1.25.0; use default_tags
+  # nodes = [
+  #  {name="", namespace="", identifier_type="", identifier="", tags = [["tag1", "value1"]]},
+  #  {name="", namespace="", identifier_type="", identifier="", tags = [["tag1", "value1"]]},
+  #]
+  #
+  # nodes = [
+  #  {name="node1", namespace="", identifier_type="", identifier=""},
+  #  {name="node2", namespace="", identifier_type="", identifier=""},
+  #]
+  #
+  ## Extra tags to be added to the output metric
+  ## Note: this syntax is needed temporarily due to a bug
+  ##   in the toml parser. Once fixed, the default_tags can be
+  ##   defined inline like tags.
+  # [inputs.opcua.group.default_tags]  # The default_tags for node1
+  #   tag1 = "value1"
+  # [inputs.opcua.group.default_tags]  # The default_tags for node2
+  #   tag1 = "value1"
+
+  ## Enable workarounds required by some devices to work correctly
+  # [inputs.opcua.workarounds]
+    ## Set additional valid status codes, StatusOK (0x0) is always considered valid
+    # additional_valid_status_codes = ["0xC0"]
+```
+
+## Node Configuration
+
+An OPC UA node ID may resemble: "ns=3;s=Temperature". In this example:
+
+- ns=3 is indicating the `namespace` is 3
+- s=Temperature is indicting that the `identifier_type` is a string and `identifier` value is 'Temperature'
+- This example temperature node has a value of 79.0
+To gather data from this node enter the following line into the 'nodes' property above:
+
+```text
+{field_name="temp", namespace="3", identifier_type="s", identifier="Temperature"},
+```
+
+This node configuration produces a metric like this:
+
+```text
+opcua,id=ns\=3;s\=Temperature temp=79.0,quality="OK (0x0)" 1597820490000000000
+```
+
+## Group Configuration
+
+Groups can set default values for the namespace, identifier type, and
+tags settings.  The default values apply to all the nodes in the
+group.  If a default is set, a node may omit the setting altogether.
+This simplifies node configuration, especially when many nodes share
+the same namespace or identifier type.
+
+The output metric will include tags set in the group and the node.  If
+a tag with the same name is set in both places, the tag value from the
+node is used.
+
+This example group configuration has two groups with two nodes each:
+
+```toml
+  [[inputs.opcua.group]]
+  name="group1_metric_name"
+  namespace="3"
+  identifier_type="i"
+  tags=[["group1_tag", "val1"]]
+  nodes = [
+    {name="name", identifier="1001", tags=[["node1_tag", "val2"]]},
+    {name="name", identifier="1002", tags=[["node1_tag", "val3"]]},
+  ]
+  [[inputs.opcua.group]]
+  name="group2_metric_name"
+  namespace="3"
+  identifier_type="i"
+  tags=[["group2_tag", "val3"]]
+  nodes = [
+    {name="saw", identifier="1003", tags=[["node2_tag", "val4"]]},
+    {name="sin", identifier="1004"},
+  ]
```

Comment:
I would give an example config with the correct supported tag notation. Maybe update the example a bit so you can show a group with shorthand nodes (without tags) and a group with the long notations with tags.